python-api/src/api/llm/router.py

```
@@ -0,0 +1,449 @@
+
+@router.post(
+    "/chat/schema",
+    response_model=ChatCompletionResponseSchema,
+    summary="Structured output chat completion",
+    status_code=status.HTTP_200_OK,
+)
+@inject
+async def structured_chat_completion(
+    body: Annotated[ChatCompletionRequestSchema, Body()],
+    consumer: InjectConsumer,
+    container: InjectContainer,
```

Ruff (0.15.2)

[warning] Unused function argument: `container`

(ARG001)